<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recordsMgmt</class>
 <widget class="QWidget" name="recordsMgmt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(66, 82, 5)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>AccontNumber</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Check No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit/Debit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountNumber">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Description">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Date">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkNo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit_debit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveRecords">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Next Records</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayRecords">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dsiplay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitMainRecords">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetCurretnInput">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>reset Current Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Balance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="previousBalance">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
